index.html

```diff
@@ -349,6 +329,62 @@ <h2 class="footer-head">
         id="footer-img"
       />
     </footer>
+    <div class="modal">
+      <div class="modal_child">
+        <div>
+          <img
+            src="./Icons/Disabled.svg"
+            alt="desktop modal close"
+            id="desk-modal-x"
+          />
+        </div>
+        <div class="modal_picture">
+          <img
+            src="./Icons/modal_close.png"
+            alt="close button"
+            id="modal_close"
+          />
+        </div>
+        <div class="mid_section">
+          <h1 class="modal_h1">Keeping track of hundreds of components</h1>
```

Comment:
- [ ] You did a very good job working  on design, pages look clean. But I can see more space between elements: 

Your design  | Figma design
------------- | -------------
<img width="296" alt="Screenshot 2022-10-11 at 21 36 55" src="https://user-images.githubusercontent.com/77038610/195172474-5bbef296-f17e-468f-a2a0-c819854d2a03.png"> |  <img width="195" alt="Screenshot 2022-10-11 at 21 36 22" src="https://user-images.githubusercontent.com/77038610/195172373-ab6b4d54-6a20-4ed9-ae27-cb81b6356c04.png">

Please make sure that popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins).